test/ClangImporter/pcm-emit-direct-cc1-mode.swift

```diff
@@ -2,6 +2,9 @@
 // RUN: %empty-directory(%t)
 // RUN: %swift-frontend -emit-pcm -direct-clang-cc1-module-build -only-use-extra-clang-opts -module-name script -o %t/script.pcm %S/Inputs/custom-modules/module.modulemap -Xcc %S/Inputs/custom-modules/module.modulemap -Xcc -o -Xcc %t/script.pcm -Xcc -fmodules -Xcc -triple -Xcc %target-triple -Xcc -x -Xcc objective-c -dump-clang-diagnostics 2> %t.diags.txt
 
+// Sometimes returns a 1 exit code with no stdout or stderr or even an indication of which command failed.
+// XFAIL: OS=linux-gnu
```

Comment:
@ian-twilightcoder, this pull [broke this test on the Android CI too](https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android/6131/console): would you please submit a followup pull modifying this to `// XFAIL: OS=linux-gnu, OS=linux-android, OS=linux-androideabi`?

Reply:
Sorry about that, https://github.com/apple/swift/pull/70523